reference-architectures/sandboxes/sandbox-modules/README.md

```diff
@@ -0,0 +1,57 @@
+# Sandbox Projects
+
+## Data Model
+
+### Deployment
+
```

Comment:
Add something like: "Each document in Firestore represents a Sandbox deployment. The following table describes the fields of the document".

Otherwise, it's not clear whether the table applies to Terraform variables or the documents.